pdns/dnsdistdist/meson-dist-script.sh

```diff
@@ -25,6 +25,8 @@ echo Running autoreconf -vi so distfile is still usable for autotools building
 # Run autoconf for people using autotools to build, this creates a configure sc
 autoreconf -vi
 rm -rf "$MESON_PROJECT_DIST_ROOT"/autom4te.cache
+echo Updating the version of the Rust library to ${BUILDER_VERSION}
+"$MESON_SOURCE_ROOT"/../../builder-support/helpers/update-rust-library-version.py "$MESON_PROJECT_DIST_ROOT"/dnsdist-rust-lib/rust/Cargo.toml dnsdist-rust ${BUILDER_VERSION}
```

Comment:
What about when building wit autotools? Are we going to deprecate that actively?

Reply:
I don't think we expect to build dist tarballs with autotools anymore, right?

Reply:
Right, we might even want to consider erroring in the autotools dist target.